db/migrate/20171205104430_create_decidim_votings_simulated_votes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateDecidimVotingsSimulatedVotes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :decidim_votings_simulated_votes do |t|
+      t.references :decidim_user, foreign_key: true
+      t.references :decidim_votings_voting, foreign_key: true , index: { name: 'index_simulated_votes_on_voting' }
+      t.integer :status, default: 0
+      t.string :voter_identifier
+      t.timestamps
```

Comment:
I think here should be a simulation id field, to allow the user to vote in different simulations in the same voting.